SpywareFilter/sections/general_extensions.txt

```diff
@@ -26,6 +26,9 @@ spletnik.ru#%#//scriptlet('set-local-storage-item', '/^_ym/', '$remove$')
 spletnik.ru#%#//scriptlet('set-local-storage-item', 'adrcid', '$remove$')
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176429
```

Comment:
This rule fixes the issue:
`@@||googletagmanager.com/gtm.js$domain=newton.by`

Reply:
no, it doesn't help, because script trying to access `yaCounter22339861`
<img width="1239" alt="Снимок экрана 2024-04-10 в 14 32 11" src="https://github.com/AdguardTeam/AdguardFilters/assets/11395678/766f43e7-41d2-4639-9076-345ae2138670">

Reply:
Please try 
```
newton.by#%#//scriptlet('set-constant', 'yaCounter22339861', 'emptyObj')
newton.by#%#//scriptlet('set-constant', 'yaCounter22339861.reachGoal', 'noopFunc')
newton.by#%#//scriptlet('set-constant', 'ga', 'noopFunc')
```

Reply:
If works - add as the main solution. @@ rule(s) - for platforms without scriptlets.